src/com/google/enterprise/adaptor/prebuilt/RegexFilter.java

```diff
@@ -102,28 +103,19 @@ public String toString() {
     }
   };
 
-  /**
-   * Which collections of keys/values to search.  Metadata, params, or both.
-   */
-  private enum Corpora {
+  // TODO (bmj): remove this and use the one in MetadataTransforms.
```

Comment:
Is that in this CL or planned for later? Need javadoc at some point, including a throws IllegalArgumentException tag for the from method.